contents/blog/ab-testing-guide-for-engineers.md

```diff
@@ -0,0 +1,205 @@
+---
+date: 2023-07-04
+title: "A software engineer's guide to A/B testing"
```

Comment:
Suggestion:

- An engineer's guide to running successful A/B tests

Reply:
Some variations...

- A/B tests that work: A software engineer's guide
- A software engineer's guide to A/B tests that work

You might be right about your original.